src/preferences/colorpaletteeditor.cpp

```diff
@@ -146,7 +146,9 @@ void ColorPaletteEditor::initialize(
     m_resetPalette = paletteName;
     QString saveName = paletteName;
 
-    for (const ColorPalette& palette : mixxx::PredefinedColorPalettes::kPalettes) {
+    const auto& kPalettes = mixxx::predefinedcolorpalettes::get();
```

Comment:
A free get() function in a namespace looks weird. This is an indicator that the Namespace is an object.

Reply:
well, its effectively a singleton. So its up to interpretation whether it should be modeled as a class with a static member function or a namespace with a free function. they're basically equivalent, though I find the namespace more elegant.